### PR TITLE
Adjust bytecode verification, ACI usage

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: governance-aepp/src/utils/settings.js [hunk 1]
```diff
@@ -27,7 +14,7 @@ export default {
     nodeUrl: 'https://mainnet.aeternity.io',
     middlewareUrl: 'https://mainnet.aeternity.io/mdw/',
     contractAddress: 'ct_ouZib4wT9cNwgRA1pxgA63XEUd8eQRrG8PcePDEYogBc1VYTq',
-    backendUrl: 'https://server.governance.aeternity.art',
-    // backendUrl: 'http://localhost:3001'
+    //backendUrl: 'https://server.governance.aeternity.art',
+    backendUrl: 'http://localhost:3001'
```
Comment (kenodressel) [inline governance-aepp/src/utils/settings.js:18]: revert?

File: governance-server/src/aeternity.js [hunk 2]
```diff
@@ -163,7 +122,7 @@ module.exports = class Aeternity {
 
     polls = async () => {
         return this.cache.getOrSet(["polls"], async () => {
-            const polls = (await this.contract.methods.polls(tempCallOptions)).decodedResult
+            const polls = (await this.contract.methods.polls()).decodedResult
```
Comment (kenodressel) [inline governance-server/src/aeternity.js:125]: no out of gas problems anymore?

File: governance-aepp/src/utils/settings.js (hunk 1 above)
Comment (thepiwo) [reply governance-aepp/src/utils/settings.js:18]: whoops yes

File: governance-server/src/aeternity.js (hunk 2 above)
Comment (thepiwo) [reply governance-server/src/aeternity.js:125]: doesn't seem so

**Review by kenodressel — APPROVED** [review]

**Pull request merged**